src/common/components/navigation/Menu.tsx

```diff
@@ -82,6 +101,20 @@ export interface MenuProps {
 
 export const Menu: React.FunctionComponent<MenuProps> = (props) => {
   const classes = useStyles();
+  const { url } = useRouteMatch();
+  const { pathname } = useLocation();
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const handleSetSelectedIndex = () => {
+    const activeItem = menuItems.findIndex(
+      (item) => pathname === `${url}${item.path}`
```

Comment:
With `Settings`, it does not work as `Settings` has sub-routes. Would work fine with 
```
pathname.startsWith(`${url}${item.path}`)
```